pkg/cmd/operator/sidecar.go

```diff
@@ -256,6 +293,11 @@ func (o *SidecarOptions) Run(streams genericclioptions.IOStreams, cmd *cobra.Com
 			case watch.Added, watch.Modified:
 				pod = e.Object.(*corev1.Pod)
 
+				if len(pod.Status.PodIP) == 0 {
```

Comment:
I guess it may not be needed in all cases but probably doesn't hurt to always test the same path

Reply:
NewMember reads it so it's better to wait until it's available in all cases